composer.lock

```diff
@@ -3558,25 +3554,25 @@
         },
         {
             "name": "rector/rector",
-            "version": "0.11.60",
+            "version": "dev-main",
```

Comment:
by requiring phpstan ^1.0 with minimum-stability dev, it seems we will have `"rector/rector": "dev-main"` in the composer.lock before next rector release. The site is using php ^8.0, so I think it is fine so we can catch the bug early.